src/index.md

```diff
@@ -1786,7 +1786,7 @@ Comments and questions may be sent to [guide-dev (at) openjdk.java.net](mailto:g
 :   A directory tree in the filesystem that Mercurial treats specially. This tree contains the source files and their revision history.
 
 [**trivial**]{#trivial}
-:   A change that is small, well contained, and makes no semantic changes. Typically fixing obvious typos or renaming some local identifier. Trivial changes can also be pushing an already reviewed change that was missed in an earlier push (e.g. forgot to add a file) or generated changes like an [`hg backout`](#backing-out-a-change). It's up to the author of a change to claim that the change is trivial in the RFR, and it's up to the Reviewer whether to approve such a claim. A change is only trivial if the Reviewer says so. Trivial changes does not have to wait 24 hours before being pushed, and only needs one Reviewer, even in areas where stricter rules for pushing normally applies.
+:   A change that is small, well contained, and makes no semantic changes. Typically fixing obvious typos or renaming some local identifier. Trivial changes can also be pushing an already reviewed change that was missed in an earlier push (e.g. forgot to add a file) or generated changes like an [`hg backout`](#backing-out-a-change). It's up to the author of a change to claim that the change is trivial in the RFR, and it's up to the Reviewer whether to approve such a claim. A change is only trivial if the Reviewer says so. Trivial changes doesn't have to wait 24 hours before being pushed, and only needs one Reviewer, even in areas where stricter rules for pushing normally applies.
```

Comment:
Unrelated to contractions, but I happened to notice this.... The last sentence needs to be either singular or plural. Either "Trivial changes don't have to wait..." or "A trivial change doesn't need to wait..." and similar for need/needs in the second half of the sentence.

This paragraph flips between singular and plural a couple times. I think I prefer singular. Or maybe this is better left to a separate editing pass, as this paragraph would benefit from other adjustments as well.

Reply:
I'm fine with doing more changes to this paragraph in this change. Better that than the changes not happening because it's too much work to create a new PR. In my opinion it's not as important to have an easy way to track down changes in a document like this as it is with source code.

If you use the GitHub Web UI for review you can edit the file directly here in the PR and suggest a change.

Reply:
Hm, I couldn't figure out how to edit in-place, so I just copied the text and made the edits below:

```
A change that is small, well contained, and that makes no semantic changes. Typical examples are fixing obvious typos or renaming a local identifier. A trivial change can also be pushing an already-reviewed change that was missed in an earlier push (e.g., forgot to add a file) or generated changes like an [`hg backout`](#backing-out-a-change). It's up to the author of a change to claim that the change is trivial in the RFR, and it's up to the Reviewer whether to approve such a claim. A change is trivial only if the Reviewer agrees that it is. A trivial change doesn't need to wait 24 hours before being pushed, and it only needs one Reviewer, even in areas where stricter rules for pushing normally apply.
```

Reply:
If you go to "Files changed". There are three dots in the upper right corner, click them to display a menu with the option to edit the file.

Reply:
I see that menu item, but it's disabled for me.